index.html

```diff
@@ -0,0 +1,197 @@
+        <p class="works-card-client-year paragraph">2015</p>
+    </div>
+    <p class="paragraph works-card-paragraph">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="works-card-tags">
+        <li class="works-card-tag">html</li>
+        <li class="works-card-tag">css</li>
+        <li class="works-card-tag">javascript</li>
+    </ul>
+    <div>
+    <button>See Project</button>
+    </div>
+</section>
+<!--Card works 2 ends-->
+
+<!--Card works 3 starts-->
+<section class="works-card">
+    <img src="./assets/Snapshoot Portfolio (2).png" alt="Recent Work">
+    <h2 class="works-card-heading-large">Tonic</h2>
+    <div class="works-card-client">
+        <p class="paragraph">CANOPY</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-role paragraph">Back End Dev</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-year paragraph">2015</p>
+    </div>
+    <p class="paragraph works-card-paragraph">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="works-card-tags">
+        <li class="works-card-tag">html</li>
+        <li class="works-card-tag">css</li>
+        <li class="works-card-tag">javascript</li>
+    </ul>
+    <div>
+    <button>See Project</button>
+    </div>
+</section>
+<!--Card works 3 ends-->
+
+<!--Card works 4 starts-->
+<section class="works-card">
+    <img src="./assets/Snapshoot Portfolio (3).png" alt="Recent Work">
+    <h2 class="works-card-heading-large">Multi-Post<br>Stories</h2>
+    <div class="works-card-client">
+        <p class="paragraph">CANOPY</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-role paragraph">Back End Dev</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-year paragraph">2015</p>
+    </div>
+    <p class="paragraph works-card-paragraph">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="works-card-tags">
+        <li class="works-card-tag">html</li>
+        <li class="works-card-tag">css</li>
+        <li class="works-card-tag">javascript</li>
+    </ul>
+    <div>
+    <button>See Project</button>
+    </div>
+</section>
+<!--Card works 4 ends-->
+
+<!-- About Section Starts Here -->
+<section class="about">
+    <div class="headline">
+        <h2 class="headline-heading heading-large">About <br> Myself</h2>
+        <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product , feature
+            or website
+            Look
+            through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
```

Comment:
- [ ] You did a good job styling the 'About me' section :+1:, but as a project requirement, you should make sure that your design match as much as possible with the Figma template, so kindly try to follow the same background as the Figma as you can see below.

**Your design**

![Screenshot from 2023-02-16 08-10-25](https://user-images.githubusercontent.com/105117832/219374436-f9b16921-094e-4393-bd4a-f5b3fc2be3df.png)

**Figma**

![Screenshot from 2023-02-16 08-10-12](https://user-images.githubusercontent.com/105117832/219374483-28c08027-54bd-4983-a6e8-a8f972b72d9a.png)

Reply:
The design match as much as possible with the Figma template.